snap/local/runtime-helpers/bin/helper-go.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+$SNAP/snap/hooks/helper-go install
+$SNAP/snap/hooks/helper-go configure
```

Comment:
This script should be split onto two. One for install and one for configure. The scripts should be placed under snap/hooks.

Snapd calls snap/hooks/configure and snap/hooks/configure executables. We no longer have them as binaries so we instead have to place the scripts. Refer to https://snapcraft.io/docs/supported-snap-hooks